### ISSUE TITLE
[Bug Report] - My Profile>Documents: Document type displayed is not user friendly

### ISSUE THREAD

**Issue by kimmykokonut** [codeforpdx/PASS#669]
### Describe the bug:
The document 'Type' is being displayed in desktop view as the variable, so Bank Statement is being shown as 'bankStatement'.

### Expected behavior:
The table holding the list of documents should show the type in a user friendly way.

### Actual Behavior:
The type column is showing the key and not the value for the DOC_TYPES object (doc_types.js)

### Screenshots:

<img width="879" alt="Screenshot 2024-08-22 at 2 39 31 PM" src="https://github.com/user-attachments/assets/bf63e8a4-d1d1-4be3-9c77-836b453935f3">

### To Reproduce:
1. Go to 'Profile'
2. Click on 'Add Document'
3. Choose document type, expiration date and enter description in the form inputs.
4. Upload file/photo
5. Click 'Upload'
6. Page updates with new document information

### Desktop (please complete the following information):
 - OS: Mac
 - Browser Firefox
 - Version 129.0.1 (64-bit)

### Possible Fix:
Display the value instead of the key of the doc_types object.

* It would also be nice to have an 'edit' option as an action, if you want to update the description or expiration date or fix an error.
* It would be nice to flag docs that are expired as once the doc is uploaded it is difficult to easily see that information.

**Comment by joshua-cornett** [comment]
I've resolved this by consequence of working on issue #665.

- "bankStatement" becomes "Bank Statement.
- "passport" becomes "Passport"
- "driversLicense" becomes "Driver's License"
- etc

**Comment by joshua-cornett** [comment]
Actually, I'll mark this closed once my PR for #665 is merged since the pertinent changes won't take affect until then.

**Comment by andycwilliams** [comment]
Yes, as a rule we should not be marking issues as completed if the code is still in review and not yet merged. Tying an issue to a pull request should automatically close it upon completion.
